src/Microsoft.Extensions.Diagnostics.HealthChecks/CompositeHealthCheckResult.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.Extensions.Diagnostics.HealthChecks
+{
+    /// <summary>
+    /// Represents the results of multiple health checks
+    /// </summary>
+    public class CompositeHealthCheckResult
```

Comment:
Is there an interesting ToString we can do here?

Reply:
I'm hesitant to put JSON in here, but maybe we could do something.

Reply:
Yeah be careful about making it look like ToString returns some kind of legitimate info as if it's a contract that people can rely on. That's not necessarily bad, but contracts have cost, so they have to be "worth it." Else, just put some nice human-readable stuff in it. Or nothing.